OpenDreamRuntime/DreamThread.cs

```diff
@@ -107,10 +107,10 @@ public abstract class ProcState {
         public virtual void ReturnedInto(DreamValue value) {}
     }
 
-    public class DreamThread {
+    public sealed class DreamThread {
         private const int MaxStackDepth = 256;
 
-        private ProcState _current;
+        private ProcState? _current;
```

Comment:
What was causing `_current` to be null?

Reply:
Can't reproduce, probably got fixed by one of the other dozen eternia PRs.